examples/rabbitmq-transaction-consumer/rabbitmq_transaction_consumer.bal

```diff
@@ -28,7 +28,7 @@ service /transactionConsumer on
                 log:printError(
```

Comment:
Better to remove the unnecessary parenthesis in the above if clock.

Reply:
Fixed in a1b4a1a23a5c079604e6d38d21165eea2c887658